components/src/core/components/Topbar/_variables.scss

```diff
@@ -1,10 +1,14 @@
 $oxd-topbar-bg-color: #ff6500;
 $oxd-topbar-bg-gradient: linear-gradient(90deg, #ff8226 0%, #f35c17 90%);
 $oxd-topbar-border-color: #e9eaec;
-$oxd-topbar-patch-color: #f7802a;
+$oxd-topbar-patch-color: #ff8226;
```

Comment:
Have to check with the design of these colors. May be not defined under colors

Reply:
some new colors have been introduced in oxd design guide for gradients. added and updated